### PR TITLE
Translate script, add file for v0.13 upgrade

### LINKED ISSUE #56: Rewrite Guided install script in Python
Terraform is installed and it still says terraform is not available.

```bash
$ terraform --version
Terraform v0.12.24

Your version of Terraform is out of date! The latest version
is 0.13.3. You can update by downloading from https://www.terraform.io/downloads.html

$ ./scripts/00-guided-install.sh
Error: Please install Terraform v0.12.24
```

### LINKED ISSUE #89: v0.12 is not compatible with v0.13
Got this error when testing qhub on DO with Terraform v0.13.3

```
qhub/test_rendering/terraform-state$ terraform init
Initializing modules...

Initializing the backend...

Initializing provider plugins...
- Finding latest version of hashicorp/digitalocean...

Error: Failed to install providers

Could not find required providers, but found possible alternatives:

  hashicorp/digitalocean -> digitalocean/digitalocean

If these suggestions look correct, upgrade your configuration with the
following command:
    terraform 0.13upgrade .terraform/modules/terraform-state/modules/digitalocean/spaces
```

Opened this PR to resolve: https://github.com/Quansight/qhub-terraform-modules/pull/23

### PR DESCRIPTION
**Pull request by tylerpotts**
https://github.com/Quansight/qhub-terraform-modules/pull/23 will need to be merged before this code will be functional. 

After the above and this are merged, fixes #56, #89

### CODE REVIEW

File: qhub/template/{{ cookiecutter.repo_directory }}/infrastructure/state.tf [hunk 1]
```diff
@@ -22,9 +22,7 @@ terraform {
     region                      = "us-west-1" # fake aws region required by terraform
     bucket                      = "{{ cookiecutter.project_name }}-terraform-state"
     key                         = "terraform/{{ cookiecutter.project_name }}.tfstate"
-    skip_requesting_account_id  = true
```
Comment (tylerpotts) [inline qhub/template/{{ cookiecutter.repo_directory }}/infrastructure/state.tf:25]: These lines were throwing this error on `terraform apply`:

```
Error: Unsupported argument

  on state.tf line 7, in terraform:
   7:     skip_requesting_account_id  = true

An argument named "skip_requesting_account_id" is not expected here.


Error: Unsupported argument

  on state.tf line 9, in terraform:
   9:     skip_get_ec2_platforms      = true

An argument named "skip_get_ec2_platforms" is not expected here.
```

File: qhub/template/{{ cookiecutter.repo_directory }}/scripts/00-guided-install.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
+from subprocess import check_output
+from shutil import which
+from os import path, environ
+import os
+
+SUPPORTED_VERSIONS = ["v0.12", "v0.13"]
```
Comment (costrouc) [inline qhub/template/{{ cookiecutter.repo_directory }}/scripts/00-guided-install.py:6]: Change to only support v0.13

Comment (tylerpotts) [reply qhub/template/{{ cookiecutter.repo_directory }}/scripts/00-guided-install.py]: updated

File: pyproject.toml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+[tool.black]
+line-length = 88
+target-version = ['py36', 'py37', 'py38']
+include = '\.pyi?$'
+exclude = '''
+/(
+    \qhub/template/{{ cookiecutter.repo_directory }}/terraform-state/versions.tf
```
Comment (costrouc) [inline pyproject.toml:7]: remove tf files from this

**Review by costrouc — APPROVED** [review]
Moving to terraform 0.13

**Pull request merged**